modules/openy_features/openy_prgf/modules/openy_prgf_classes_listing/openy_prgf_classes_listing.module

```diff
@@ -23,7 +23,51 @@ function openy_prgf_classes_listing_form_views_exposed_form_alter(&$form, FormSt
       $form_state->setUserInput($user_input);
     }
 
+    // Change Select All labels.
+    $selectFields = ['location', 'program', 'category', 'activity'];
+    foreach ($selectFields as $selectField) {
+      if (isset($form[$selectField])) {
+        $form[$selectField]['#options']['All'] = t('All');
+      }
+      if ($selectField == 'location') {
+        $form[$selectField]['#options'] = openy_prgf_classes_listing_get_location_options();
+      }
+    }
+
     $form['#attributes']['class'][] = 'container';
     $form['#attached']['library'][] = 'openy_prgf_classes_listing/openy_classes_listing';
   }
 }
+
+/**
+ * Predefine options for locations.
+ * @return array
+ */
+function openy_prgf_classes_listing_get_location_options() {
+  $options = [];
+
+  if (!$options) {
```

Comment:
`$options` is always empty here and defined above. So, this condition doesn't make sense.